lib/i18n/tasks/data/language_names.rb

```diff
@@ -0,0 +1,202 @@
+# frozen_string_literal: true
+
+module I18n
+  module Tasks
+    module Data
+      module LanguageNames # rubocop:disable Metrics/ModuleLength
+        # Data loaded from https://github.com/grosser/i18n_data
+        # I18nData.languages.transform_keys!(&:to_sym).transform_values!{|v| v.split(";").first }
```

Comment:
@glebm or should we add `i18n_data` as a optional dependency which is only used if it is available?

Reply:
I think it's better to have it here as it might need to be expanded it in the future to handle region codes, such as `zh-TW` as "Taiwanese Mandarin"`.